### ISSUE TITLE
Adding devices and updating device names

### ISSUE THREAD

**Issue by SierraStreamsInstitute-Lab** [thosmos/riverdb#44]
We are moving to a more detailed device names, which should address the redundancy of the current naming system in RiverDB (ie YSI devices measure both dissolved oxygen and conductivity). 

Our conductivity meters would be EC-SSI-1, EC-SSI-2, etc. 

We also need to add new devices for silica, bacteria, phosphate and nitrate (if we move them out of the lab section). 

I am wondering whether, if we were to make these fixes (update device names, do turbidity medians, etc.), could we then delete all of our data currently in RiverDB and re-upload? Because the find and replace function in Excel would be a lot faster than doing a lot of these things manually. Thanks for your thoughts!

**Comment by thosmos** [comment]
Yes I think re-uploading would be faster and better

**Comment by SierraStreamsInstitute-Lab** [comment]
OK. So I just need to get you all of our updated device names to repopulate those lists?

**Comment by thosmos** [comment]
Yes and or i need to get the device editor working well enough for you to
be able to do it.

T

**Comment by SierraStreamsInstitute-Lab** [comment]
I'm also seeing that data pulled from RiverDB does not have site codes pre-2021. Do you know if that's accurate? Did we just not used to use them? Or is it another potential issue with the data downloader?

**Comment by thosmos** [comment]
The pre-2021 data was just test data.  I think the best approach is to start over from scratch, especially because you have a few parameters that have been de-activated, etc.
